MCprep_addon/addon_updater_ops.py

```diff
@@ -1284,6 +1284,10 @@ def skip_tag_function(self, tag):
 	# if 'beta' in tag.lower():
 	# 	return True
 	# ---- write any custom code above, return true to disallow version --- #
+	
+	# Ignore release candidates
+	if 'rc' in tag.lower():
+		return True
```

Comment:
Maybe to give us some wiggle room, we could do a few checks? Since we know we won't get the benefit of such code updates until *after* the next time people get an update notice once they adopt this next version.

e.g. 

```suggestion
	skip_if_present = ['rc', 'alpha', 'beta']
	for skip_name in skip_if_present:
		if skip_name in tag.lower():
			return True
```